products/wso2_MB/combined_scenarios/MBTestCases/src/main/java/ttl/Queues/QueueReceiveDefault.java

```diff
@@ -0,0 +1,80 @@
+package ttl.Queues;
+
+import javax.jms.JMSException;
+import javax.jms.MessageConsumer;
+import javax.jms.QueueConnection;
+import javax.jms.QueueSession;
+import javax.naming.NamingException;
+import javax.jms.*;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.util.Properties;
+
+/**
+ * Created by pubudup on 9/5/16.
+ */
+public class QueueReceiveDefault {
+
+    public static final String QPID_ICF = "org.wso2.andes.jndi.PropertiesFileInitialContextFactory";
```

Comment:
Can we not use a constant file and retrieve these from one place. Configs seem to be repeated.